std/internal/math/biguintcore.d

```diff
@@ -35,12 +35,12 @@ module std.internal.math.biguintcore;
 
 version(D_InlineAsm_X86)
 {
-    import std.internal.math.biguintx86;
-}
-else
-{
-    import std.internal.math.biguintnoasm;
+    static import std.internal.math.biguintx86;
 }
+static import std.internal.math.biguintnoasm;
```

Comment:
This now will lead to a bigger object size on `x86` as this file will always be imported, but `dmd` isn't smart enough to lazy load local imports anyways. Besides `if (__ctfe)` isn't even `static if`, so imports within will always be loaded.

Reply:
Yeah i guess this is the counterpart of the trick used to handle symbol conflicts.

Reply:
I was trying to think of various ways to work around this, but could not.

I wonder if there's a way to introduce a CTFE-only template, basically one that is not instantiated even if it's referred to, until the CTFE engine tries to execute it.